zqd/packet/packet.go

```diff
@@ -108,6 +108,7 @@ func (p *IngestProcess) run(ctx context.Context) error {
 	abort := func() {
 		os.RemoveAll(p.logdir)
 		os.Remove(p.space.DataPath(IndexFile))
+		os.Remove(p.space.DataPath("all.bzng"))
```

Comment:
If there is a "slurp"/zeek error (bad name on my part) `run` will not wait for an ongoing snapshot routine to complete before it aborts. Thus there is a race and an all.bzng could be written after the fact. I don't think this needs to be addressed here, but it should be addressed at some point.

Reply:
Good point. There's also crash recovery that could leave an all.bzng around and needs to be addressed (we have an issue for that).